tools/bazel.rc

```diff
@@ -55,11 +55,13 @@ build:sycl_asan --copt -DGPR_NO_DIRECT_SYSCALLS --linkopt -fPIC
 build:sycl_asan --linkopt -fsanitize=address
 build:sycl_asan --copt -DDISABLE_SKINNY=1
 
+# This configuration is for a cross-build by default.
+# For a native build on ARM, specify --cpu=arm on the command line.
 build:sycl_arm --crosstool_top=@local_config_sycl//crosstool:toolchain
 build:sycl_arm --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
 build:sycl_arm --define=using_sycl=true --define=using_trisycl=false
-build:sycl_arm --cpu=armeabi
 build:sycl_arm --incompatible_load_argument_is_label=false
+build:sycl_arm --cpu=armeabi
```

Comment:
Is this really needed? Does the order matter in that instance?

Reply:
It was accidental - feel free to revert it

Reply:
We can't really revert it unless we do another PR and another commit. If you don't have time to do it I'll just leave it as it is.